digits/dataset/images/generic/test_views.py

```diff
@@ -61,18 +61,16 @@ class BaseViewsTestWithImageset(BaseViewsTest):
     @classmethod
     def setUpClass(cls):
         super(BaseViewsTestWithImageset, cls).setUpClass()
-        cls.imageset_folder = tempfile.mkdtemp()
-        # create imageset
-        cls.test_image = create_lmdbs(cls.imageset_folder)
+        if not hasattr(BaseViewsTestWithImageset, 'imageset_folder'):
+            # Create folder and LMDBs for all test classes
+            BaseViewsTestWithImageset.imageset_folder = tempfile.mkdtemp()
+            BaseViewsTestWithImageset.test_image = create_lmdbs(BaseViewsTestWithImageset.imageset_folder)
         cls.created_datasets = []
-
     @classmethod
     def tearDownClass(cls):
         # delete any created datasets
         for job_id in cls.created_datasets:
             cls.delete_dataset(job_id)
-        # delete imageset
```

Comment:
Yeah, I'm intentionally leaking a temporary directory in this PR. They get cleaned up on reboot and I can't imagine many people use the test suite on anything other than a development machine.

I'm surprised that so many temporary folders are created, though. Do you know where that's coming from? I'm going to go ahead and merge this PR, but we can continue the discussion here and perhaps address it in a separate PR if necessary.